plugins/ibm_qradar/unit_test/test_connection_test.py

```diff
@@ -0,0 +1,57 @@
+"""Test Cases for the Qradar Connection."""
```

Comment:
@mrinehart-r7 I believe we can't leave these tests in because of our unit test runner not having credentials. I believe @blaxminarayan-r7 will be handling integration test like these in another fashion. But its good to see these were run!